quick-start.yml

```diff
@@ -65,16 +78,43 @@ services:
       - 18082:18082
       - 19092:19092
       - 19644:9644
-
-  generator:
-    image: "haianh1233/generator:latest"
-    container_name: generator
+    healthcheck:
+      test: ["CMD-SHELL", "rpk cluster health | grep -E 'Healthy:.+true' || exit 1"]
+      interval: 15s
+      timeout: 3s
+      retries: 5
+      start_period: 5s
+  
+  conduktor-gateway:
+    image: conduktor/conduktor-gateway:2.3.0
+    hostname: conduktor-gateway
+    container_name: conduktor-gateway
     environment:
-      KAFKA_BOOTSTRAP_SERVERS: "redpanda-0:9092"
+      KAFKA_BOOTSTRAP_SERVERS: redpanda-0:9092
+    ports:
+      - "8888:8888"
+    healthcheck:
+      test: curl localhost:8888/health
+      interval: 5s
+      retries: 25
+    depends_on:
+      redpanda-0:
+        condition: service_healthy
+  
+  conduktor-data-generator:
+    image: "conduktor/conduktor-data-generator:0.1"
```

Comment:
Why the `"` ?

Reply:
I think I was following the earlier use of them in the other services, not sure why they have them.
Have removed for this as not needed.